src/core/localize.rkt

```diff
@@ -23,64 +23,66 @@
                  (loop arg atype))])))))
 
 ;; Returns a list of expressions sorted by increasing local error
-(define (localize-error expr ctx)
-  (define errs (compute-local-errors expr ctx))
-  (sort
-    (reap [sow]
-          (for ([(expr err) (in-hash errs)])
-            (unless (andmap (curry = 1) err)
-              (sow (cons err expr)))))
-    > #:key (compose errors-score car)))
+(define (batch-localize-error exprs ctx)
+  (define errss
+    (if (null? exprs) empty (compute-local-errors exprs ctx)))
+  (for/list ([expr (in-list exprs)] [errs (in-list errss)])
+    (sort
+     (reap [sow]
```

Comment:
Can't this `reap [sow]` just be a `for/list` with an `#:unless` clause?

Reply:
Yes... I didn't see this before I merged it, my bad, and I don't think I'm gonna bother with another PR for now. Hopefully I see it next time we work on this file.